tests/unit/test_kraft.py

```diff
@@ -165,4 +166,4 @@ def test_ready_to_start(charm_configuration, base_state: State):
     # Only the internal users should be created.
     assert "admin-password" in next(iter(state_out.secrets)).latest_content
     assert "sync-password" in next(iter(state_out.secrets)).latest_content
-    assert state_out.unit_status == ActiveStatus()
+    assert state_out.unit_status == ActiveStatus(), logger.error(f"{state_out.unit_status=}")
```

Comment:
**question**: I don't mind the statement after the assertion, but I am curious: what issue did you have here? pytest should properly display the two operands if they are not equal